.github/workflows/docs.yml

```diff
@@ -0,0 +1,36 @@
+name: Docs CI
+
+on:
+  push:
+    branches:
```

Comment:
Do we want to run this for tags as well?

Speaking to the one who sorted this out for our own docs, it seems that by default there is no option in GitHub Pages to easily select other tags, as there is for readthedocs. He made a change to the Sphinx template (not sure how) to add this, but we are using a different template here. If this interests you, I will talk to him about how we can 'port' this over.

You can probably navigate to the documentation for other tags even without the above change.

Reply:
For now, I am not yet thinking of keeping several versions of the documentation. It is so late behind the code that the main point is to improve it and keep it up-to-date with the code evolution. So updating on push is perfect, a tag is not related to any modification of the sources. Note that in the final version, only the `master` branch should be kept, the `sphinx_automation` branch is only there to check that it's properly working.

For the future, one could think of keeping a doc version corresponding to each PyAT release, so your ideas are welcome ! For now, the idea is that whatever PyAY release or commit you are running, the later documentation is the best.